pages/learn/getting-started.md

```diff
@@ -25,42 +25,188 @@ If you don't already have a {{ $names.company.lower }} account, make sure to [si
+
+```shell
+$ {{ $names.company.lower }} apps
+ID    APP NAME   DEVICE TYPE          ONLINE DEVICES DEVICE COUNT
+98264 First-App  {{ $device.name }}   0              0
+```
+
+__Note:__ See all the commands available with {{ $names.company.lower }} CLI by running `{{ $names.company.lower }} help`
+
+<!-- This is language specific -->
+{{import "getting-started/deployingCode"}}
+
+__Note:__ You may also use git to deploy code to a device. If you wish to deploy via git, see the instructions [here][gitdocs].
+
+Now to deploy this code to all device(s) in the application, use the `{{ $names.company.lower }} push First-App` command replacing `First-App` with the name of your application. Ensure you are in the root of the project directory before issuing this command or specify the `--source` option to provide an alternate location of the project directory.
+
+```shell
+$ {{ $names.company.lower }} push First-App
+```
+
+This command will package up and push the code from the local directory to the {{ $names.company.lower }} builders, where it will be compiled, built and deployed to every device in the application fleet.
+
+You'll know your code has been successfully compiled and built when our friendly unicorn mascot appears in your terminal:
+
+```shell
+[main]     Successfully built d5f1de77fad3
+[Info]     Uploading images
+[Success]  Successfully uploaded images
+[Success]  Release successfully created!
+[Info]     Release: f4e3925bf7d32226365225e1b7201b90 (id: 89693)
+[Info]     ┌─────────┬────────────┬────────────┐
+[Info]     │ Service │ Image Size │ Build Time │
+[Info]     ├─────────┼────────────┼────────────┤
+[Info]     │ main    │ 205.13 MB  │ 1 second   │
+[Info]     └─────────┴────────────┴────────────┘
+[Info]     Build finished in 7 seconds
+			    \
+			     \
+			      \\
+			       \\
+			        >\/7
+			    _.-(6'  \
+			   (=___._/` \
+			        )  \ |
+			       /   / |
+			      /    > /
+			     j    < _\
+			 _.-' :      ``.
+			 \ r=._\        `.
+			<`\\_  \         .`-.
+			 \ r-7  `-. ._  ' .  `\
+			  \`,      `-.`7  7)   )
+			   \/         \|  \'  / `-._
+			              ||    .'
+			               \\  (
+			                >\  >
+			            ,.-' >.'
+			           <.'_.''
+			             <'
+```
+
+Your application will then be downloaded and executed by all the devices you have connected in your application fleet. The first push is slower to deploy, but all subsequent pushes are much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:
+
+<img alt="Service download progress" src="/img/common/device/download-progress.png" width="100%">
+
+<!-- This is language/project specific -->
 {{import "getting-started/postPush"}}
 
-{{import "getting-started/deviceUrl"}}
```

Comment:
What was the reason for removing these types of partials? I think this partial could definitely be re-used in an FAQ and perhaps elsewhere where one just wants to reference how to enable a URL.

Reply:
> I've moved the generic content into the same file and only using imports for variable content. I don't have a strong opinion on this and can revert; it just made my life easier when editing this and makes it more obvious which content is intended to be variable.

Thanks and quoting myself :), this was more for my benefit in editing and no good rationale. Happy to add some back in where they could be reused like this one you suggest.

Reply:
yeah it makes sense to only add things if they are useful in other places. When i first wrote this, the Getting started was the first use of the partials, so i tried to write most things for reuse in the docs and elsewhere, but agree it only makes sense if its actually useful and reusuable.